### ISSUE TITLE
z-test instead of t-test in the group sequential implementation

### ISSUE THREAD

**Issue by daryadedik** [zalando/expan#118]
In the group sequential implementation is used z-test instead of t-test and sample standard deviation is used instead of population standard deviation. If the population standard deviation is unknown it would be more reasonable to use t-test instead of z-test.

**Comment by shansfolder** [comment]
@jbao Is there a reason you use z-test(instead of t-test as Darya described above) when deciding early stopping in the group sequential method? :)

**Comment by daryadedik** [comment]
just a note&question: when the sample size is large, the sample standard deviation can be used as an approximation of population standard deviation, but then how large the sample size should be and what is the problem with switching to t-test instead?

**Comment by jbao** [comment]
You're absolutely right, if you compare L92 and L99 in `early_stopping.py` you would notice that z-score is used to compare with a threshold, but the interval is calculated by the percentiles of the t-distribution. I agree it probably makes sense to use the t-statistics.

**Comment by shansfolder** [comment]
thank you @jbao , we will plan and update the method. ;)
